specutils/io/default_loaders/wcs_fits.py

```diff
@@ -1,36 +1,44 @@
 import logging
-import os
+import warnings
 
 from astropy import units as u
 from astropy.io import fits
-from astropy.wcs import WCS
-from astropy.modeling import models, fitting
+from astropy.wcs import WCS, _wcs
+from astropy.modeling import models
+from astropy.utils.exceptions import AstropyUserWarning
 
+import numpy as np
 import shlex
 
 from ...spectra import Spectrum1D
-from ..registers import data_loader
+from ..registers import data_loader, custom_writer
 
 __all__ = ['wcs1d_fits_loader', 'wcs1d_fits_writer', 'non_linear_wcs1d_fits']
 
 
 def identify_wcs1d_fits(origin, *args, **kwargs):
-    # check if file can be opened with this reader
     # args[0] = filename
+    # check if filename conforms to naming convention for writer
+    if origin == 'write':
```

Comment:
We might want the identifiers to also check what type of file object `args[0]` is in the case that it might be an already existing `HDUList`.